css-typed-om/Overview.bs

```diff
@@ -2892,6 +2903,52 @@ depending on the property they came from:
         return "currentcolor".
     2. Otherwise, return the result of serializing the <<color>> value.
 
+: 'margin'
+::
+    1. Let |values| initially be the empty [=list=].
+    2. If 'margin-bottom' does not specify ''0'' for it's initial value, 
+        serialize 'margin-bottom' and append it to |values|.
+    3. If 'margin-left' does not specify ''0'' for it's initial value, 
+        serialize 'margin-left' and append it to |values|.
+    4. If 'margin-right' does not specify ''0'' for it's initial value, 
+        serialize 'margin-right' and append it to |values|.
+    5. If 'margin-top' does not specify ''0'' for it's initial value, 
+        serialize 'margin-top' and append it to |values|.
+    6. If 'margin' is set to the ''inherit'' keyword, append "inherit" to |values|.
+    7. Return the result of concatenating all the items in |values|,
+        separated by " " (U+0020 SPACE).
```

Comment:
Checked other browsers and fixed it to include collapsing. (Copied the Chrome implementation here)